# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"POT-Creation-Date: 2016-07-17 19:58+0200\n"
"PO-Revision-Date: 2016-07-19 01:45+0200\n"
"Last-Translator: \n"
"Language-Team: \n"
"Language: es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"
"X-Generator: Poedit 1.8.8\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:63
msgid "Update failed!"
msgstr "¡La actualización ha fallado!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:63
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4130
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4140
msgid "Could not connect to %s"
msgstr "No se puede conectar a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:258
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:260
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:262
msgid "Operating System not supported: %s"
msgstr "Sistema Operativo no soportado: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:277
msgid "Copy old appdata to new folder failed!"
msgstr ""
"¡Ha fallado la copia de datos de aplicación antigua a la carpeta nueva!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:277
msgid "Error: def win_pre1_check_app_dir()"
msgstr "Error: def win_pre1_check_app_dir()"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:288
msgid "Could not create app_dir: %s"
msgstr "No se puede crear app_dir: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:315
msgid "Select User-ID failed!"
msgstr "¡Ha fallado la selección de la ID de usuario!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:380
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:383
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:386
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:389
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:392
msgid "Error: '%s' not found!"
msgstr "Error: '%s' ¡No encontrado!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:409
msgid ""
"Could not remove lock file.\n"
"File itself locked because another oVPN Client instance running?"
msgstr ""
"No se pudo eliminar el archivo de bloqueo.\n"
"¿Está el archivo bloqueado por otra instancia de cliente de oVPN "
"ejecutándose?"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:454
msgid ""
"Creating API-DIRS\n"
"%s \n"
"%s \n"
"%s \n"
"%s \n"
"%s failed!"
msgstr "¡La creación de API-DIRS %s %s %s %s %s ha fallado!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:456
msgid "Creating config Folders failed"
msgstr "Ha fallado la creación de las carpetas de configuración"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:683
msgid "Error: def read_options_file"
msgstr "Error: def read_options_file"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:683
msgid "Read config file failed!"
msgstr "¡Ha fallado la lectura del archivo de configuración!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:816
msgid ""
"Error reading your Interfaces from netsh.exe. Contact support@ovpn.to with "
"Error-ID:\n"
"ADAPTERS[1]=(%s)\n"
msgstr ""
"Error al leer sus Interfaces desde netsh.exe. Contacte con support@ovpn.to "
"con la Error-ID:\n"
"ADAPTERS[1]=(%s)\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:851
msgid ""
"Could not read your Network Interfaces! Please install OpenVPN or check if "
"your TAP-Adapter is really enabled and driver installed."
msgstr ""
"¡No se pudo leer las Interfaces de Red! Por favor, instale OpenVPN o "
"compruebe que su Adaptador TAP está habilitado e instalado el controlador."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:882
msgid "No OpenVPN TAP-Windows Adapter found!"
msgstr "¡No se ha encontrado el Adaptador OpenVPN TAP-Windows!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:896
msgid ""
"Multiple TAPs found!\n"
"\n"
"Please select your TAP Adapter!"
msgstr "¡Múltiples TAPs encontrados! ¡Por favor, seleccione su Adaptador TAP!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:912
msgid ""
"No OpenVPN TAP-Adapter found!\n"
"Please install openVPN!\n"
"URL1: %s\n"
"URL2: %s"
msgstr ""
"¡No se ha encontrado ningún Adaptador TAP!\n"
" ¡Por favor, instale openVPN!\n"
" URL1: %s\n"
" URL2: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:917
msgid "Invalid characters in '%s'"
msgstr "Carácteres inválidos en '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:933
msgid "Choose your External Network Adapter!"
msgstr "¡Seleccione su Adaptador de Red Externo!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:951
msgid "No Network Adapter found!"
msgstr "¡No se encontró ningún adaptador de red!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:966
msgid "Please select your User-ID!"
msgstr "¡Por favor, seleccione su ID de Usuario!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1013
msgid "Disconnect %s"
msgstr "Desconectar %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1017
msgid "Connect to %s"
msgstr "Conectar a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1025
msgid "Remove AutoConnect: %s"
msgstr "Eliminar AutoConnect: %s"

# To Set accepts multiple variants like:
# poner
# ajustar
# fijar
# colocar
# etc...
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1029
msgid "Set AutoConnect: %s"
msgstr "Establecer AutoConnect %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1044
msgid "[enabled]"
msgstr "[habilitado]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1046
msgid "[disabled]"
msgstr "[deshabilitado]"

# If Server is plural, it would be 'de los Servidores'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1047
msgid "Load Server Information %s"
msgstr "Cargar información del Servidor %s"

# Same as above: ' de los Servidores'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1060
msgid "Set Server-View Size [%sx%s]"
msgstr "Ajuste tamaño de vista del Servidor [%sx%s]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1067
msgid "Update every: %s seconds"
msgstr "Actualizar cada: %s segundos"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1085
msgid "Change DNS"
msgstr "Cambiar DNS"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1159
msgid "Set Primary DNS"
msgstr "Establezca DNS Primario"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1169
msgid "Set Secondary DNS"
msgstr "Establezca DNS Secundario"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1330
msgid "Checking for Updates!"
msgstr "¡Comprobando actualizaciones!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1332
msgid "Downloading Configurations..."
msgstr "Descargando configuraciones..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1334
msgid "Requesting Certificates..."
msgstr "Solicitando certificados..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1336
msgid "Please wait... Certificates requested from backend! (%s)"
msgstr ""
"Solicitado los certificados desde el back-end (%s), por favor, espere..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1338
msgid "Downloading Certificates..."
msgstr "Descargando certificados..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1340
msgid "Extracting Configs and Certs..."
msgstr "Extrayendo configuraciones y certificados..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1344
msgid "Disconnected! Have a nice and anonymous day!"
msgstr "¡Desconectado! ¡Que tenga un bonito y anónimo día!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1358
msgid "Connecting to %s"
msgstr "Conectando a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1364
msgid "Testing connection to %s"
msgstr "Comprobando conexión a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1369
msgid "Connection to %s unstable or failed!"
msgstr "Conexión inestable o fallada a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1383
msgid "Connected to %s [%s]:%s (%s) [ %s ] (%s / %s ms)"
msgstr "Conectado a %s [%s]:%s (%s) [ %s ] (%s / %s ms)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1546
msgid " [ disconnect ]"
msgstr "[ desconectar ]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1589
msgid "Close Servers"
msgstr "Cerrar Servidores"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1592
msgid "Servers"
msgstr "Servidores"

# Also:
# Salir de la cuenta
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1602
msgid "Close Account"
msgstr "Cerrar Cuenta"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1604
msgid "Account"
msgstr "Cuenta"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1614
msgid "Close Settings"
msgstr "Cerrar Ajustes"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1616
msgid "Settings"
msgstr "Ajustes"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1629
msgid "About"
msgstr "Acerca de"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1637
msgid "Quit"
msgstr "Salir"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1696
msgid "Error: Update Timeout"
msgstr "Error: Tiempo expirado para actualización"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1696
msgid ""
"Update took too long...aborted!\n"
"Please retry in few minutes..."
msgstr ""
"¡La actualización ha llevado demasiado tiempo...abortado!\n"
"Por favor, reinténtelo en unos minutos..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1707
msgid "Certificates and Configs updated!"
msgstr "¡Los certificados y configuraciones han sido actualizadas!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1707
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1720
msgid "oVPN Update OK!"
msgstr "¡Actualización de oVPN correcta!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1710
msgid "Extraction failed!"
msgstr "¡La extracción ha fallado!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1710
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1712
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1715
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1717
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1725
msgid "Error: def inThread_timer_check_certdl"
msgstr "Error: def inThread_timer_check_certdl"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1712
msgid "Failed to download certificates"
msgstr "Falló la descarga de certificados"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1715
msgid "Failed to request certificates!"
msgstr "¡Falló la solicitud de certificados!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1717
msgid "Failed to download configurations!"
msgstr "¡Falló la descarga de configuración!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1720
msgid "No update needed!"
msgstr "¡No es necesaria ninguna actualización!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1725
msgid "Failed to check for updates!"
msgstr "¡Falló la comprobación de actualizaciones!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1970
msgid "oVPN Server - %s"
msgstr "Servidor oVPN - %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2042
msgid "oVPN Server [ %s ]"
msgstr "Servidor oVPN [ %s ]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2178
msgid "oVPN Account - %s"
msgstr "Cuenta oVPN  - %s"

# Alternative: 'Cargar información de cuenta'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2198
msgid "Settings -> Updates: 'Load Account Info' [disabled]"
msgstr "Ajustes -> Actualizaciones: 'Cargar ajustes de cuenta' [deshabilitado]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2200
msgid "No data loaded! Retry in few seconds..."
msgstr "¡No se ha cargado ningún dato! Reinténtelo en unos segundos..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2297
msgid "oVPN Settings - %s"
msgstr "Ajustes oVPN - %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2330
msgid " Security "
msgstr " Seguridad"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2347
msgid " Options "
msgstr "  Opciones"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2361
msgid " Updates "
msgstr " Actualizaciones"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2371
msgid "Restore Firewall Backups\n"
msgstr "Restaurar copias de seguridad del cortafuegos\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2373
msgid " Backups "
msgstr "Copias de seguridad"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2381
msgid "Use Windows Firewall (default: ON) "
msgstr "Usar Cortafuegos de Windows (por defecto: encendido)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2411
msgid "TAP-Adapter block outbound (default: OFF)"
msgstr "Bloque Saliente de Adaptador de TAP (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2442
msgid "Block Internet on Disconnect or Quit (default: ON)"
msgstr "Bloquear Internet al desconectar o cerrar (por defecto: encendido)"

# Instead 'FW' it can be 'Cortafuegos'
# already very long, client supposed to know what is FW.
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2470
msgid "Disable FW question on Quit (default: OFF)"
msgstr "Deshabilitar pregunta acerca de FW al salir (por defecto: encendido)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2498
msgid "Clear Rules on Connect (default: OFF)"
msgstr "Reglas claras al conectar (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2518
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2548
msgid "Could not export Windows Firewall Backup!"
msgstr "¡No se pudo exportar copia de seguridad de cortafuegos de Windows!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2518
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2548
msgid "Error: Windows Firewall Backup failed"
msgstr "Error: Falló la copia de seguridad del cortafuegos de Windows"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2528
msgid "Backup on Start / Restore on Quit (default: OFF)"
msgstr ""
"Copia de seguridad al Inicio / Restaurar al Salir (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2558
msgid "DNS Leak Protection (default: ON)"
msgstr "Protección de fuga de DNS (por defecto: encendido)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2588
msgid "Disable '%s' on Disconnect (default: OFF)"
msgstr "Deshabilitar '%s' al Desconectar (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2626
msgid "Update Configs on Start (default: OFF)"
msgstr "Actualizar configuraciones al Iniciar (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2651
msgid "Load Account Info (default: OFF)"
msgstr "Cargar información de la cuenta (por defecto: apagado)"

# Not clear if Server is single or plural
# Used single here.
# In plural would be 'de los servidores' instead 'del servidor' only.
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2685
msgid "Load Server Info (default: OFF)"
msgstr "Cargar información del Servidor (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2720
msgid "Debug Mode (default: OFF)"
msgstr "Modo de depuración (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2736
msgid "Debug Mode Enabled"
msgstr "Modo de depuración habilitado"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2736
msgid ""
"Logfile:\n"
"'%s'"
msgstr ""
"Archivo de registro:\n"
" '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2745
msgid "Select Network Adapter"
msgstr "Seleccione Adaptador de Red"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2756
msgid "Use IPv4 Entry Server with Exit to IPv4 (standard)"
msgstr "Selección: Servidor entrante con salida a IPv4 (estándar)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2759
msgid "Use IPv4 Entry Server with Exits to IPv4 + IPv6"
msgstr "Selección: Servidor entrante con salida a IPv4 + IPv6"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2784
msgid "Change App Theme (default: ms-windows)"
msgstr "Cambiar el tema de la aplicación (por defecto: ms-windows)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2817
msgid "Change App Font Size (default: 9)"
msgstr "Cambiar tamaño de fuente de la aplicación (por defecto: 9)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2863
msgid "Change Language"
msgstr "Cambiar idioma"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2899
msgid "Disable About and Quit on Connection (default: ON)"
msgstr "Deshabilitar Acerca de y Salir al conectar (por defecto: encendido)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2920
msgid "Normal Config Update"
msgstr "Actualización de la configuración normal"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2929
msgid "Forced Config Update"
msgstr "Actualización forzada de la configuración"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2938
msgid "Reset API-Login"
msgstr "Reiniciar API-Login"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3172
msgid "Please wait!"
msgstr "¡Por favor, espere!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3172
msgid "Update is running."
msgstr "Actualización en curso."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3248
msgid "Error: Certs not found!"
msgstr "Error: ¡No se encontraron los certificados!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3248
msgid "Files missing: '%s'"
msgstr "Archivos faltantes: '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3276
msgid "Could not start Windows Firewall!"
msgstr "¡No se pudo iniciar el cortafuegos de Windows!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3276
msgid "Error: def inThread_spawn_openvpn_process"
msgstr "Error: def inThread_spawn_openvpn_process"

# Not sure what it means
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3587
msgid "Error: DNS restore 2nd"
msgstr "Error: Restaurar DNS secundario"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3587
msgid "Error: Restore Secondary DNS to %s failed."
msgstr "Error: Restaurar DNS secundario si falló %s."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3590
msgid "Error: DNS restore 1st"
msgstr "Error: Restaurar DNS primario"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3590
msgid "Error: Restore Primary DNS to %s failed."
msgstr "Error: Restaurar DNS primario si falló %s."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3653
msgid "def win_netsh_read_dns_to_backup: failed!"
msgstr "def win_netsh_read_dns_to_backup: ¡fallado!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3686
msgid ""
"Error:\n"
"SSL Root Certificate for %s not loaded %s"
msgstr ""
"Error:\n"
"Certificado Raíz SSL para %s no cargado %s"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3686
msgid "Error: SSL CA Cert #1"
msgstr "Error: SSL CA Cert #1"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3689
msgid ""
"Error:\n"
"Invalid SSL Root Certificate for %s in:\n"
"'%s'\n"
"hash is:\n"
"'%s'\n"
"but should be '%s'"
msgstr ""
"Error:\n"
"Certificado Raíz SSL inválido para %s en:\n"
"'%s'\n"
"hash es:\n"
"'%s'\n"
"pero debería ser '%s'"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3689
msgid "Error: SSL CA Cert #2"
msgstr "Error: SSL CA Cert #2"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3692
msgid ""
"Error:\n"
"SSL Root Certificate for %s not found in:\n"
"'%s'!"
msgstr "Error: ¡No se encontró Certificado Raíz SSL para %s en: '%s'!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3692
msgid "Error: SSL CA Cert #3"
msgstr "Error: SSL CA Cert #3"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3881
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3905
msgid "Error: netsh.exe not found!"
msgstr "Error: ¡No se encontró netsh.exe!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3920
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3944
msgid "Error: route.exe not found!"
msgstr "Error: ¡No se encontró route.exe!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3960
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3972
msgid "ipconfig.exe not found!"
msgstr "¡No se encontró ipconfig.exe!"

# In case if it is an installer.
# otherwise it is 'Ajustes oVPN.to'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4007
msgid "oVPN.to Setup"
msgstr "Asistente de configuración oVPN.to"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4008
msgid "Enter your oVPN.to Details"
msgstr "Introduzca los detalles de su oVPN.to"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4017
msgid "User-ID:"
msgstr "ID de usuario:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4026
msgid "API-Key:"
msgstr "Clave API:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4130
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4140
msgid "Error: Update failed"
msgstr "Error: ¡La actualización ha fallado!"

# 'Servidores' in case of plural
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4183
msgid "Server Window Size"
msgstr "Tamaño ventana Servidor"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4187
msgid "Width (pixel):"
msgstr "Ancho (pixeles):"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4191
msgid "Height (pixel):"
msgstr "Alto (pixeles):"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4195
msgid ""
"Enter width and height\n"
"\n"
"Leave blank for default"
msgstr ""
"Introduzca ancho y alto\n"
"\n"
"Dejar vacío para valores por defecto"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4242
msgid "Load Data every X seconds"
msgstr "Cargar Datos cada X segundos"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4246
msgid "seconds:"
msgstr "segundos:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4293
msgid "Switched to IPv4+6"
msgstr "Conmutado a IPv4+6"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4293
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4310
msgid ""
"Changed Option:\n"
"\n"
"Use 'Forced Config Update' to get new configs!\n"
"\n"
"You have to join 'IPv6 Beta' on https://%s to use any IPv6 options!"
msgstr ""
"Opción cambiada:\n"
"\n"
"¡Use 'Actualización forzada de la configuración' para obtener nuevas "
"configuraciones!\n"
"\n"
"¡Tiene que ingresar en 'IPv6Beta' en https://%s para usar cualquier opción "
"IPv6!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4310
msgid "Switched to IPv6+4"
msgstr "Conmutado a IPv6+4"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4360
msgid "No API-Key!"
msgstr "¡No hay Clave API!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4360
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4391
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4402
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4414
msgid "Error: def API_REQUEST"
msgstr "Error: def API_REQUEST"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4391
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4414
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4695
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4752
msgid "Invalid User-ID or API-Key or Account expired!"
msgstr "¡ID de usuario inválida o Clave API o Cuenta expirada!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4402
msgid "API requests on: %s failed!"
msgstr "¡Las solicitudes de la API fallaron en: %s!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4515
msgid "Error: def move_appdata"
msgstr "Error: def move_appdata"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4515
msgid "Move Userdata to new folder: '%s'"
msgstr "Mover Datos de usuario a carpeta nueva: '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4551
msgid "Could not read Servers Remote-IP:Port from config: %s"
msgstr ""
"No se pudo leer la IP Remota de los Servidores:Puerto de la configuración: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4560
msgid "Could not read Servers Protocol from config: %s"
msgstr "No se pudo leer Protocolo de los Servidores desde la configuración: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4572
msgid "Could not read Servers Cipher from config: %s"
msgstr ""
"No se pudo leer el Cifrado de los Servidores desde la configuración: %s"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4598
msgid "Error: def load_ovpn_server"
msgstr "Error: def load_ovpn_server"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4598
msgid "Load openVPN Server failed"
msgstr "Falló la carga del Servidor openVPN"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4774
msgid "Platform '%s' not supported"
msgstr "Plataforma '%s' no soportada"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4774
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4825
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4829
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4913
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4966
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4969
msgid "Error!"
msgstr "¡Error!"

# If setup refers to some windows installer.
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4790
msgid ""
"openVPN Setup downloaded and hash verified OK!\n"
"\n"
"Please start setup from file:\n"
"'%s'\n"
"\n"
"Verify GPG with:\n"
"'%s'"
msgstr ""
"¡Descargado ajustes de openVPN con verificación hash exitosa!\n"
"\n"
"Por favor, inicie el asistente de configuración desde el archivo:\n"
"'%s'\n"
"\n"
"Verifique GPG con:\n"
"'%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4792
msgid ""
"openVPN Setup downloaded but hash verify failed!\n"
"Please install openVPN!\n"
"URL1: %s\n"
"URL2: %s"
msgstr ""
"¡Descargado ajustes de openVPN pero falló la verificación hash!\n"
"¡Por favor, instale openVPN!\n"
"URL1: %s\n"
"URL2: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4799
msgid ""
"Install OpenVPN %s (%s) (%s)\n"
"\n"
"Starting download (~1.8 MB) from:\n"
"'%s'\n"
"to\n"
"'%s'\n"
"\n"
"Please wait..."
msgstr ""
"Instale OpenVPN %s (%s) (%s)\n"
"\n"
"Comenzando descarga (~1.8MB) desde:\n"
"'%s'\n"
"a\n"
"'%s'\n"
"\n"
"Por favor, espere..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4799
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4865
msgid "Setup: openVPN"
msgstr "Asistente de configuración: openVPN"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4825
msgid ""
"Invalid File hash: %s !\n"
"localhash = '%s'\n"
"but should be = '%s'"
msgstr ""
"¡Hash de archivo inválido: %s !\n"
"localhash = '%s'\n"
"pero debería ser = '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4829
msgid "Failed remove file: %s"
msgstr "No se pudo eliminar el archivo: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4865
msgid ""
"OpenVPN not found!\n"
"\n"
"Please select openvpn.exe on next window!\n"
"\n"
"If you did not install openVPN yet: click cancel on next window!"
msgstr ""
"¡No se encontró OpenVPN!\n"
"\n"
"¡Por favor, seleccione openvpn.exe en la siguiente ventana!\n"
"\n"
"¡Si aún no ha instalado OpenVPN haga click en cancelar en la siguiente "
"ventana!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4866
msgid "Select openvpn.exe or Cancel to install openVPN"
msgstr "Seleccione openvpn.exe o Cancelar para instalar openVPN"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4913
msgid ""
"WARNING! Failed to verify files in\n"
"'%s'\n"
"\n"
"Uninstall openVPN and restart oVPN Client Software!\n"
"\n"
"Or install openVPN from URL:\n"
"%s[debug self.LAST_FAILED_CHECKFILE = '%s']"
msgstr ""
"¡Advertencia! Falló la verificación de archivos en\n"
"'%s'\n"
"¡Desinstale openVPN y reinicie el cliente oVPn!\n"
"\n"
"O instale openVPN desde URL:\n"
"%s[debug self.LAST_FAILED_CHECKFILE = '%s']"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4918
msgid "Could not detect openVPN Version!"
msgstr "¡No se pudo detectar la versión de openVPN!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4942
msgid "Could not find openVPN"
msgstr "¡No se encuentra openVPN!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4966
msgid "Invalid Hash: '%s'! is '%s' != '%s'"
msgstr "¡Hash no válido: '%s'! es '%s' !='%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4969
msgid "Invalid content '%s' in '%s'"
msgstr "Contenido no válido '%s' en '%s'"

# If arch means architecture!
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4986
msgid "arch '%s' not supported!"
msgstr "¡arquitectura '%s' no soportada!"

# also: 'Cerrar cliente oVPN.to'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5194
msgid "Quit oVPN.to Client"
msgstr "Salir del cliente oVPN.to"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5198
msgid "Do you really want to quit?"
msgstr "¿Está seguro de salir?"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5264
msgid "Firewall Settings"
msgstr "Configuración del cortafuegos"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5268
msgid ""
"Restore previous firewall settings?\n"
"\n"
"Press 'YES' to restore your previous firewall settings!\n"
"Press 'NO' to set profiles to 'blockinbound,blockoutbound'!"
msgstr ""
"¿Restaurar ajustes previos del cortafuegos?\n"
"\n"
"¡Presione 'SI' para restaurar sus ajustes previos del cortafuegos!\n"
"¡Presione 'NO' para fijar los perfiles a 'blockinbound,blockoutbound'!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5271
msgid ""
"Allow outgoing connection to internet?\n"
"\n"
"Press 'YES' to set profiles to 'blockinbound,allowoutbound'!\n"
"Press 'NO' to set profiles to 'blockinbound,blockoutbound'!"
msgstr ""
"¿Permitir conexión saliente hacia Internet?\n"
"\n"
"¡Presione 'SI' para fijar los perfiles a 'blockinbound,allowoutbound'!\n"
"¡Presione 'NO' para fijar los perfiles a 'blockinbound,allowoutbound'!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5326
msgid "Error"
msgstr "Error"

#~ msgid "Security Settings\n"
#~ msgstr "Ajustes de seguridad\n"

#~ msgid "Options\n"
#~ msgstr "Opciones\n"

#~ msgid "Updates\n"
#~ msgstr "Actualizaciones\n"

#~ msgid "Change App Theme"
#~ msgstr "Cambiar tema de la App"

# "Preferencias" is also widely used and valid :)
#~ msgid "Options"
#~ msgstr "Opciones"

#~ msgid "IPv6 Options"
#~ msgstr "Opciones IPv6"

#~ msgid "Select: IPv4 Entry Server with Exit to IPv4 (standard)"
#~ msgstr "Selección: Servidor entrante con salida a IPv4 (estándar)"

#~ msgid "Select: IPv4 Entry Server with Exits to IPv4 + IPv6"
#~ msgstr "Selección: Servidor entrante con salida a IPv4 + IPv6"

#~ msgid "Firewall"
#~ msgstr "Cortafuegos"

#~ msgid "Restore Firewall Backups"
#~ msgstr "Restaurar copias de seguridad de Cortafuegos"

#~ msgid "Disable FW question on Quit (default: ON)"
#~ msgstr ""
#~ "Deshabilitar pregunta acerca de FW al salir (por defecto: encendido)"
